src/integrationTest/resources/gradle_builds/build_1.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'com.github.qwazer.markdown-confluence'
+}
+
```

Comment:
Am I right that confluence server must run locally on that port for a proper testing?

Reply:
Yep, but there's an image provided by the author that you can just run and then run the integration tests. By default when running the gradle build, only the unit tests will be executed. To run the integration tests (integrationTest task), you need to run the container and then run the Gradle task.